src/control/build/config_test.go

```diff
@@ -0,0 +1,191 @@
+//
+// (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package build
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/daos-stack/daos/src/control/common/test"
+	"github.com/pkg/errors"
```

Comment:
Oops, I'd guess this is an error from my patch, but we normally import Go packages in three clusters:
- Built-in packages
- External packages
- DAOS packages

Minor aesthetic issue, we can correct it later if you don't need to push again.

```suggestion
	"github.com/pkg/errors"

	"github.com/daos-stack/daos/src/control/common/test"
```

Reply:
sounds good.  I will fix it if I push again. I'm hoping I do not have to do so.